packages/truffle-core/lib/testing/Assert.sol

```diff
@@ -1184,497 +986,214 @@ library Assert {
+            result (bool) - The result.
     */
-    function _tag(bool value, string memory tag) internal pure returns (string memory) {
-        string memory nstr = _ltoa(value);
-        return _tag(nstr, tag);
+    function balanceIsZero(address a, string memory message) internal returns (bool result) {
+        return AssertBalance.balanceIsZero(a, message);
     }
 
     /*
-        Function: _appendTagged(string)
-
-        Append a tagged value to a string.
-
-        Params:
-            tagged (string) - The tagged value.
-            str (string) - The string.
-
-        Returns:
-            result (string) - "str (tagged)"
-    */
-    function _appendTagged(string memory tagged, string memory str) internal pure returns (string memory) {
-
-        bytes memory taggedB = bytes(tagged);
-        bytes memory strB = bytes(str);
-
-        uint sl = strB.length;
-        uint tl = taggedB.length;
-
-        bytes memory newB = new bytes(sl + tl + 3);
-
-        uint i;
-        uint j;
-
-        for (i = 0; i < sl; i++)
-            newB[j++] = strB[i];
-        newB[j++] = ' ';
-        newB[j++] = '(';
-        for (i = 0; i < tl; i++)
-            newB[j++] = taggedB[i];
-        newB[j++] = ')';
-
-        return string(newB);
-    }
+        Function: balanceIsNotZero
 
-    /*
-        Function: _appendTagged(string, string)
+        Assert that the balance of an account 'A' is not zero.
 
-        Append two tagged values to a string.
+        : A.balance != 0
 
         Params:
-            tagged0 (string) - The first tagged value.
-            tagged1 (string) - The second tagged value.
-            str (string) - The string.
+            A (address) - The first address.
+            message (string) - A message that is sent if the assertion fails.
 
         Returns:
-            result (string) - "str (tagged0, tagged1)"
+            result (bool) - The result.
     */
-    function _appendTagged(string memory tagged0, string memory tagged1, string memory str) internal pure returns (string memory) {
-
-        bytes memory tagged0B = bytes(tagged0);
-        bytes memory tagged1B = bytes(tagged1);
-        bytes memory strB = bytes(str);
-
-        uint sl = strB.length;
-        uint t0l = tagged0B.length;
-        uint t1l = tagged1B.length;
-
-        bytes memory newB = new bytes(sl + t0l + t1l + 5);
-
-        uint i;
-        uint j;
-
-        for (i = 0; i < sl; i++)
-            newB[j++] = strB[i];
-        newB[j++] = ' ';
-        newB[j++] = '(';
-        for (i = 0; i < t0l; i++)
-            newB[j++] = tagged0B[i];
-        newB[j++] = ',';
-        newB[j++] = ' ';
-        for (i = 0; i < t1l; i++)
-            newB[j++] = tagged1B[i];
-        newB[j++] = ')';
-
-        return string(newB);
+    function balanceIsNotZero(address a, string memory message) internal returns (bool result) {
+        return AssertBalance.balanceIsNotZero(a, message);
     }
-
 }
```

Comment:
Future work for this file: we can use the ABIs for the various Assert type libraries and autogenerate Assert.sol